scripts/workflow_helper.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
```

Comment:
Please add a comment with some explanation
Maybe we should add in other places.

Reply:
Done. Thanks